iothub/device/samples/PnpDeviceSamples/Thermostat/Program.cs

```diff
@@ -7,10 +7,11 @@
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Provisioning.Client;
```

Comment:
Are you planning to do temperature controller sample changes in a separate PR?

Reply:
No, i'll it to this PR, once I get a green from the certification flow.

Reply:
👍